apps/charterafrica/src/components/Tool/Contributors.js

```diff
@@ -0,0 +1,40 @@
+import { RichTypography } from "@commons-ui/core";
+import { Avatar, AvatarGroup, Box } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+const Contributors = React.forwardRef(function Contributors(props, ref) {
+  const { list, sx, title } = props;
+  if (!list?.length) {
+    return null;
+  }
+  return (
+    <Box ref={ref} sx={sx}>
+      {list?.length ? (
+        <Box sx={{ p: 2 }}>
+          <RichTypography color="neutral.dark" variant="h4Small">
+            {title}
+          </RichTypography>
+          <AvatarGroup sx={{ justifyContent: { xs: "center", sm: "right" } }}>
+            {list.map((item) => (
+              <Avatar key={item.id} alt={item.name} src={item.avatarUrl} />
+            ))}
+          </AvatarGroup>
+        </Box>
+      ) : null}
```

Comment:
```suggestion
      <Box sx={{ p: 2 }}>
        <RichTypography color="neutral.dark" variant="h4Small">
          {title}
        </RichTypography>
        <AvatarGroup sx={{ justifyContent: { xs: "center", sm: "right" } }}>
          {list.map((item) => (
            <Avatar key={item.id} alt={item.name} src={item.avatarUrl} />
          ))}
        </AvatarGroup>
      </Box>
```

You've already checked `list.length` before `return`.